### PR TITLE
Update Example .travis.yml in CI Guide

### PR DESCRIPTION
**Pull request by mtmckenna**
Installing applesimutils seems to be required to have builds pass in Travis.

### CODE REVIEW

File: docs/Guide.RunningOnCI.md [hunk 1]
```diff
@@ -68,6 +68,8 @@ env:
   - NODE_VERSION=stable
 
 install:
+- brew tap wix/brew
+- brew install --HEAD applesimutils
```
Comment (LeoNatan) [inline docs/Guide.RunningOnCI.md]: Please do no longer use `—HEAD`. There are proper versions now, which should be stable.

**Comment by mtmckenna** [comment]
Based on feedback from @LeoNatan , I removed the `--HEAD` argument from the `brew` command. I also removed `--HEAD` from the same command I found elsewhere in the docs.

I left the `--HEAD` in `scripts/install.ios.sh` because I didn't want this pull request to modify any code. However, perhaps it makes sense to make the change to `.install.ios.sh` as well in a different commit?

File: docs/Guide.RunningOnCI.md (hunk 1 above)
Comment (mtmckenna) [reply docs/Guide.RunningOnCI.md]: Thank you! Updated the pull request.

**Comment by mtmckenna** [comment]
Hello! Anything I should do to help get this one merged in? Thank you!

**Pull request merged**